cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/DurableLogInspectCommand.java

```diff
@@ -245,13 +239,38 @@ Predicate<OperationInspectInfo> getConditionTypeFromUser() {
         return predicate;
     }
 
+    private void getValueOrRange(List<Predicate<OperationInspectInfo>> predicates, String conditionTpe) {
+        String op;
+        op = getStringUserInput("Search operation based on: [value/range]");
+        if (op.equals("value")) {
+            long in = getLongUserInput("Enter valid " + conditionTpe);
+            if (conditionTpe.equals(SEQUENCE_NUMBER)) {
+                predicates.add(a -> a.getSequenceNumber() == in);
+            }
+            if (conditionTpe.equals(SEGMENT_ID)) {
+                predicates.add(a -> a.getSegmentId() == in);
+            }
+            if (conditionTpe.equals(OFFSET)) {
+                predicates.add(a -> a.getOffset() == in);
+            }
+            if (conditionTpe.equals(LENGTH)) {
+                predicates.add(a -> a.getLength() == in);
+            }
+            if (conditionTpe.equals(ATTRIBUTES)) {
+                predicates.add(a -> a.getAttributes() == in);
+            }
+        } else {
```

Comment:
If something other than "value" is used they will get "range". This is a serious problem if someone for example types "Value" or "value ".